tests/DocBlock/TypeExpressionTest.php

```diff
@@ -372,6 +374,24 @@ public static function provideParseInvalidExceptionCases(): iterable
 
         yield ['((unclosed_parenthesis)'];
 
+        yield ['|vertical_bar_start'];
+
+        yield ['&ampersand_start'];
+
+        yield ['~tilde_start'];
+
+        yield ['vertical_bar_end|'];
+
+        yield ['ampersand_end&'];
+
+        yield ['tilde_end~'];
```

Comment:
```suggestion
```

Reply:
same as in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8210#discussion_r1766276289